### DISCUSSION TITLE
Apple silicon build

### DISCUSSION THREAD

**Discussion by DiaDiaDia99** [fangkehou-team/BingGPT#8] [General]
Are you planning to update the app for arm Macs?

**Comment by lony2003** [comment]
Well, actually I have planned to provide arm64 build for three platforms, like #5 . But I still have no clue about the login problem and I don't want to release the new version with just a bump of version number. But if you really want, I can release a newer version, just don't expect to have a functional login fature, like #6 .

**Comment by lony2003** [comment]
New version with arm64 build was released! Please download the latest version in releases page.
